gtsam/slam/tests/testInitializePose3.cpp

```diff
@@ -75,8 +75,14 @@ NonlinearFactorGraph graph2() {
   g.add(BetweenFactor<Pose3>(x0, x1, pose0.between(pose1), noiseModel::Isotropic::Precision(6, 1.0)));
   g.add(BetweenFactor<Pose3>(x1, x2, pose1.between(pose2), noiseModel::Isotropic::Precision(6, 1.0)));
   g.add(BetweenFactor<Pose3>(x2, x3, pose2.between(pose3), noiseModel::Isotropic::Precision(6, 1.0)));
-  g.add(BetweenFactor<Pose3>(x2, x0, Pose3(Rot3::Ypr(0.1,0,0.1), Point3()), noiseModel::Isotropic::Precision(6, 0.0))); // random pose, but zero information
-  g.add(BetweenFactor<Pose3>(x0, x3, Pose3(Rot3::Ypr(0.5,-0.2,0.2), Point3(10,20,30)), noiseModel::Isotropic::Precision(6, 0.0))); // random pose, but zero informatoin
+  // random pose, but zero information
+  g.add(BetweenFactor<Pose3>(x2, x0,
```

Comment:
format

Reply:
I did format and that is what I got.

Reply:
Hmm. This is very strange...

Reply:
Did a refactor so that the formatting is more consistent.